File: app/src/main/java/cat/deim/asm/myapplication/model/BikeModel.java
package cat.deim.asm.myapplication.model;

public class BikeModel {
    private String uuid;
    private String marca;
    private int activo;

    public BikeModel(String uuid, String marca, int activo) {
        this.uuid = uuid;
        this.marca = marca;
        this.activo = activo;
    }

    public String getUuid() { return uuid; }
    public String getMarca() { return marca; }
    public int getActivo() { return activo; }

    public void setUuid(String uuid) { this.uuid = uuid; }
    public void setMarca(String marca) { this.marca = marca; }
    public void setActivo(int activo) { this.activo = activo; }
}

File: app/src/main/java/cat/deim/asm_34/patinfly/model/BikeModel.kt
package cat.deim.asm.myapplication.model

import com.google.gson.annotations.SerializedName
import java.util.*

data class BikeModel(
    val uuid: UUID,
    @SerializedName("bike_type") val bikeType: String,
    @SerializedName("is_available") val isAvailable: Boolean,
    @SerializedName("battery_level") val batteryLevel: Int,
    @SerializedName("creation_date") val creationDate: Date
)
